tests/Sitemap/SitemapFactoryTest.php

```diff
@@ -93,48 +109,78 @@ public function testCreateContentSitemapWithHit()
         $this->invokeReflectionMethod('createContentSitemap');
     }
 
-    protected function setUp()
+    public function testSitemapFactoryException(): void
     {
-        parent::setUp();
+        $committeeRepository = $this->createMock(CommitteeRepository::class);
```

Comment:
you can use this method too (not a request changes)
`createConfiguredMock(CommitteeRepository::class, ['findSitemapCommittees' => []])`